packages/app/src/cli/services/context/breakdown-extensions.ts

```diff
@@ -45,13 +45,21 @@ export async function extensionsIdentifiersDeployBreakdown(options: EnsureDeploy
   }
   remoteExtensionsRegistrations: RemoteExtensionRegistrations
 }> {
-  const remoteExtensionsRegistrations = await options.developerPlatformClient.appExtensionRegistrations({
+  let remoteExtensionsRegistrations = await options.developerPlatformClient.appExtensionRegistrations({
     id: options.appId,
     apiKey: options.appId,
     organizationId: '0',
   })
 
   const extensionsToConfirm = await ensureExtensionsIds(options, remoteExtensionsRegistrations.app)
```

Comment:
It's also really unclear why `ensureExtensionIds` is responsible for migration, the name doesn't suggest much. That cannot be helped in this PR I guess though